cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerSetupService.java

```diff
@@ -619,10 +618,6 @@ private ApiConfig removeRemoteParcelRepos() {
         return new ApiConfig().name("remote_parcel_repo_urls").value("");
     }
 
-    private ApiConfig disableGoogleAnalytics() {
```

Comment:
you deleted this. probably a wrong merge. please put it back

Reply:
done